dist/main.js

```diff
@@ -0,0 +1,105 @@
+const display = new Renderer
+const module = new APIManager
+
+const loadPage = async function () {
+    display.renderPage('home')
+    const countryName = await module.getCurrentCountry()
+    module.countryName = countryName
+    await module.getStats(countryName, '', '')
+    module.createChart()
+    await module.getNews()
+    display.renderNews(module.news)
+}
+
+$('#menu-bar').on('click', '#menu-button', function () {
+    display.renderMenu()
+})
+
+$('#menu-bar').on('click', '#delete-menu', function () {
+    display.renderOfMenu()
+})
+
+$('#search-button').on('click', async function () {
+    const countryName = $('#country-input').val()
+    $('#country-input').val('')
+    module.countryName = countryName
+    try {
+        await module.getStats(countryName, '', '')
+        display.createChartTemplate()
+        module.createChart()
+    }
+    catch{
+        display.renderEror()
+    }
+
+})
+
+$('#home').on('click', function () {
+    display.renderPage('home')
+    loadPage()
+})
+
+$('#covid-chat').on('click', function () {
+    display.renderPage('calculatur')
+})
+
+$('#stats').on('click', async function () {
+    display.renderPage('stats')
+    await module.getWorldStats()
+    await module.getCountryStats()
+    display.renderStats(module.worldStats)
+    module.createCarthForCharPage()
+    module.getUsersInfoFromDB()
+})
+
+$('#news-bar').on('click', function () {
+    module.news.splice(5)
+    display.renderNewsPage(module.news)
+})
+
+$('#about').on('click', function () {
+    display.renderPage('about')
+})
+
+$('#page-content').on('click', '.answer', function () {
+    const text = $(this).text()
+    const next = module.calculaturQue(text)
+    display.renderChat(text, "user-msg")
+    setTimeout(function () {
+        display.renderChat(next[0], 'domain-msg')
+        if (next[0] === 'We got results! Please submit your info for answers') {
+            display.renderUserForm(false)
+        } else {
+            if (next[0] === 'We got results! Please submit your info and date of exposer for answers') {
+                display.renderUserForm(true)
+            } else {
+                display.renderQue(next[1], next[2])
+            }
+        }
+    }, 700)
+})
+
+$('#page-content').on('click', '.submit-user', function () {
+    const name = $(this).closest('div').find('.name-input').val()
```

Comment:
Do you need traversal here? Or are these always the only ones on the page? Maybe you can directly select them...?